app/scripts/directives/addConfigToApplication.js

```diff
@@ -26,12 +26,35 @@
     var ctrl = this;
     var humanizeKind = $filter('humanizeKind');
 
+    var checkContainerForRef = function(container) {
```

Comment:
Tested, I think we might need an additional message option:

![screen shot 2017-10-11 at 11 31 37 am](https://user-images.githubusercontent.com/280512/31450615-e1964aba-ae77-11e7-81b4-4c545f5691dc.png)

It says "There are no applications in this project", but that isn't true in this case.  I've just already added this secret to both of the apps in my project.

Reply:
@benjaminapetersen what about "There is not any application with containers to add secret to"

Reply:
We might want to show all apps in the list, but show an error with a disabled save button when the user tries to pick one that already has the secret. Otherwise it might seem like a bug that their app isn't showing up, and it's not obvious why.